### PR TITLE
Better solution to forceSelectedID

### PR DESCRIPTION
**Pull request by mlunoe**

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @rcorral

File: src/Dropdown/Dropdown.js [hunk 1]
```diff
@@ -290,6 +280,8 @@ export default class Dropdown extends Util.mixin(BindMixin) {
       );
     }
 
+    let selectedID = props.forceSelectedID || state.selectedID;
```
Comment (kennyt) [inline src/Dropdown/Dropdown.js]: maybe we can abstract this logic out into a method called `getSelectedID`

**Comment by jfurrow** [comment]
I like this, but I'm not crazy about the prop `forceSelectedID`. I think we should use the word "persistent," like `persistentID` or `persistentItemID` perhaps?

File: src/Dropdown/Dropdown.js (hunk 1 above)
Comment (rcorral) [inline src/Dropdown/Dropdown.js]: +1

**Comment by rcorral** [comment]
I like the name @jfurrow came up with!

File: src/Dropdown/Dropdown.js (hunk 1 above)
Comment (mlunoe) [inline src/Dropdown/Dropdown.js]: Good point!

Comment (mlunoe) [inline src/Dropdown/Dropdown.js]: done

**Comment by mlunoe** [comment]
Changed name!

**Pull request merged**